interop/run.py

```diff
@@ -1,35 +1,45 @@
-from enum import Enum
-import os
-from prettytable import PrettyTable
-import random
-import shutil
-import subprocess
-import string
+import os, random, shutil, subprocess, string, logging, sys, argparse
 from typing import List
 from termcolor import colored
+from enum import Enum
+from prettytable import PrettyTable
 
 import testcases
 
-class TestResult(Enum):
-  SUCCEEDED = 1
-  FAILED = 2
-  UNSUPPORTED = 3
-
 # add your QUIC implementation here
 IMPLEMENTATIONS = { # name => docker image
-  "quicgo": "quicgo"
+#  "quicgo": "quicgo"
+  "h2o-qns": "h2o-qns"
 }
+
 TESTCASES = [ 
   testcases.TestCaseTransfer(),
   testcases.TestCaseRetry(),
   testcases.TestCaseResumption(),
 ]
 
+def get_args():
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-l', '--loglevel', default='WARNING',
```

Comment:
Do we really need log levels? I think of logging here as a binary thing: If the test succeeds, we don't need to print logs. If it fails, we always want to print logs (and save the log files from the containers).

Reply:
Nobody _needs_ to use this. I was actually going to pass down the log levels into the endpoint docker images so that folks could use it as they want. I can make it so that simply specifying --debug (or -d) means setting loglevel to DEBUG, which is what most use cases will be. WDYT?